com.microsoft.mrtk.core/Tests/TestUtilities/RuntimeTestUtilities.cs

```diff
@@ -100,23 +100,49 @@ public static void InstallTextMeshProEssentials()
 #endif
         }
 
+        private static bool testPaused;
         /// <summary>
-        /// Used for debugging. Pauses the test until the enter key is pressed.
+        /// Used for debugging. Pauses the test until the dialog is cleared.
         /// </summary>
-        public static IEnumerator WaitForEnterKey()
+        public static IEnumerator PauseTest()
         {
-            Debug.Log(Time.time + " | Press Enter...");
-#if ENABLE_INPUT_SYSTEM
-            while (!Keyboard.current[Key.Enter].wasPressedThisFrame)
+#if UNITY_EDITOR
```

Comment:
I believe this will still be defined in batch-mode tests, right? Might be worth an additional runtime check for `isBatchMode` just to make sure any Pauses left in don't trigger in batch mode.

Reply:
Done